Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosItemTests.cs

```diff
@@ -139,13 +139,15 @@ public async Task CreateDropItemTest()
             Assert.IsFalse(diagnostics.IsGoneExceptionHit());
             Assert.IsFalse(string.IsNullOrEmpty(diagnostics.ToString()));
             Assert.IsTrue(diagnostics.GetClientElapsedTime() > TimeSpan.Zero);
+            Assert.IsFalse(response.Diagnostics.FailedRequestCount() > 0);
```

Comment:
nit: Using Assert.AreEqual will include both values in the error message making it easier to troubleshoot if it fails.
```suggestion
            Assert.AreEqual(0, response.Diagnostics.FailedRequestCount());
```